packages/plugins/integration-make-com/src/lib/make-com.service.ts

```
@@ -1,142 +1,462 @@
-
-		if (!tenantId) {
-			throw new NotFoundException('Tenant ID not found in request context');
+	async saveOAuthCredentials(credentials: { clientId: string; clientSecret: string }): Promise<void> {
+		try {
+			const tenantId = RequestContext.currentTenantId();
+			if (!tenantId) {
+				throw new NotFoundException('Tenant ID not found in request context');
+			}
+
+			// Find the integration for the current tenant
+			let integrationTenant = await this.integrationTenantService.findOneByOptions({
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'integrationTenant' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)